docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-completion.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: completion
+permalink: reference/mesheryctl/system/completion
+type: reference
```

Comment:
Same way here...

Reply:
This one too, will push the commit today